src/dbos-executor.ts

```diff
@@ -58,6 +58,11 @@ export interface DBOSConfig {
   readonly application?: object;
   readonly dbClientMetadata?: any;
   readonly debugProxy?: string;
+  readonly http?: {
```

Comment:
The other option is to remove this completely and ask for a decorator if you want to configure CORS at all, like what we do for the custom body parser or custom middleware.  Personally, I think that's better because I prefer having information in code than in YAML files, but either way works.

Reply:
In this particular case, I suspect that the CORS whitelist may be dependent on deployment, and for that reason I don't think you'd want to bake it into the code.  (Though it could be that the application provides code that checks the application section of yaml.)

Reply:
Body parser of course is determined by the code.